libs/ngx-charts-on-fhir/src/lib/fhir-chart/fhir-chart-configuration.service.ts

```diff
@@ -22,6 +22,21 @@ type MergedDataLayer = {
   providedIn: 'root',
 })
 export class FhirChartConfigurationService {
+  today_date_annotation: ChartAnnotation = {
+    type: 'line',
+    borderColor: '#FF900D',
+    borderWidth: 3,
+    display: true,
+    label: {
+      display: true,
+      content: 'Today',
+      position: 'start',
+      color: '#FF900D',
+      backgroundColor: '#FAFAFA',
+    },
+    scaleID: 'x',
+    value: new Date().getTime(),
+  };
```

Comment:
Maybe we should provide this via DI so apps can override it if they want to? We could do it like the ones in fhir-mapper-options.ts (but keep it in this file) where the InjectionToken specifies a default implementation. What do you think?